### DISCUSSION TITLE
Packaging exe with pyinstaller fails to start

### DISCUSSION THREAD

**Discussion by Chenlin2020-center** [israel-dryer/ttkbootstrap#318] [Q&A]
Launch the exe file in cmd，prompt "_tkinter.TclError: invalid command name "::msgcat::mcmset"   "

**Comment by TheDemonGuard** [comment]
> Launch the exe file in cmd，prompt "_tkinter.TclError: invalid command name "::msgcat::mcmset" "

I am not able to reproduce the issue with the latest version (1.14.6).
Is it possible to try with the latest version?
If the issue still persists, please provide steps/instructions to reproduce the issue along with a demo/sample script.
